docs/griptape-framework/structures/config.md

```diff
@@ -34,6 +34,23 @@ agent = Agent(
 )
 ```
 
+#### Anthropic
+
+The [Anthropic Structure Config](../../reference/griptape/config/structure_config.md#griptape.config.structure_config.AnthropicStructureConfig) provides default Drivers for Anthropic's APIs.
+
+!!! info
+    Anthropic does not provide an embeddings API which means you will need to use another service for embeddings.
+    The `AnthropicStructureConfig` defaults to using `VoyageAiEmbeddingDriver` which integrates with [VoyageAI](https://www.voyageai.com/), the service used in Anthropic's [embeddings documentation] (https://docs.anthropic.com/claude/docs/embeddings).
```

Comment:
can you add an example of how a user would overload the default?

Reply:
Linked to example.